test/NVQPP/bug_qubit.cpp

```diff
@@ -18,7 +18,7 @@ struct simple_x {
   void operator()() __qpu__ {
     cudaq::qubit q;
     x(q);
-    mz(q);
+    auto result = mz(q);
```

Comment:
Should we throw in a `[[maybe_unused]]` on these examples?

Reply:
If we want to do that, I think we should open a new PR for that as there are multiple tests that do this same thing.